test/examples/examples.jl

```diff
@@ -114,6 +114,63 @@
+            @test_nowarn_mod trixi_include(@__MODULE__,
+                                           joinpath(examples_dir(), "fluid",
+                                                    "dam_break_2d_surface_tension.jl"),
+                                           tspan=(0.0, 0.1))
+            @test sol.retcode == ReturnCode.Success
+            @test count_rhs_allocations(sol, semi) == 0
+        end
+
+        @trixi_testset "fluid/deformation_sphere_2d.jl" begin
+            @test_nowarn_mod trixi_include(@__MODULE__,
+                                           joinpath(examples_dir(), "fluid",
+                                                    "deformation_sphere_2d.jl"))
+            @test sol.retcode == ReturnCode.Success
+            @test count_rhs_allocations(sol, semi) == 0
+        end
+
+        @trixi_testset "fluid/deformation_sphere_3d.jl" begin
+            @test_nowarn_mod trixi_include(@__MODULE__,
+                                           joinpath(examples_dir(), "fluid",
+                                                    "deformation_sphere_3d.jl"),
+                                           tspan=(0.0, 1.0))
+            @test sol.retcode == ReturnCode.Success
+            @test count_rhs_allocations(sol, semi) == 0
+        end
+
+        @trixi_testset "fluid/falling_water_spheres_2d.jl" begin
+            @test_nowarn_mod trixi_include(@__MODULE__,
+                                           joinpath(examples_dir(), "fluid",
+                                                    "falling_water_spheres_2d.jl"),
+                                           tspan=(0.0, 0.2), sphere1_center=(0.5, 0.2),
+                                           sphere2_center=(1.5, 0.2))
+            @test sol.retcode == ReturnCode.Success
+            @test count_rhs_allocations(sol, semi) == 0
+        end
+
+        @trixi_testset "fluid/falling_water_spheres_3d.jl" begin
+            @test_nowarn_mod trixi_include(@__MODULE__,
+                                           joinpath(examples_dir(), "fluid",
+                                                    "falling_water_spheres_3d.jl"),
+                                           tspan=(0.0, 0.1), fluid_particle_spacing=0.01) [
+                r"┌ Info: The desired tank length in x-direction .*\n",
+                r"└ New tank length in x-direction.*\n",
+                r"┌ Info: The desired tank length in y-direction .*\n",
+                r"└ New tank length in y-direction.*\n",
+            ]
+            @test sol.retcode == ReturnCode.Success
+            @test count_rhs_allocations(sol, semi) == 0
+        end
+
+        @trixi_testset "fluid/wetting_water_spheres_2d.jl" begin
+            @test_nowarn_mod trixi_include(@__MODULE__,
+                                           joinpath(examples_dir(), "fluid",
+                                                    "wetting_water_spheres_2d.jl"),
+                                           tspan=(0.0, 0.1))
```

Comment:
Since these example can easily be unstable with different resolutions, could you please include the exact file here without changing parameters, so that we can guarantee that the examples are working?
As I said earlier, the examples should be made cheaper anyway.